examples/computer_vision_techniques/mask_disk.py

```diff
@@ -42,6 +43,9 @@
 palette = aia.plot_settings['cmap']
 palette.set_bad('black')
 
+###############################################################################
```

Comment:
I don't feel this comment explains enough, why is it negative, why this range and etc.